reference-stores/nuxt/components/search/SearchFilters.vue

```diff
@@ -12,19 +12,12 @@
       @click="showFilterDrawer = true"
     >
       <span class="text-sm font-medium text-gray-700">Filters</span>
-      <svg
+      <!-- eslint-disable vue/no-v-html -->
```

Comment:
This is another good opportunity to use:

```
<!-- eslint-disable-next-line vue/no-v-html -->
```

Reply:
@NWRichmond I don't think this works.  It only works for elements that stay on a single line.
<img width="456" alt="Screen Shot 2022-05-05 at 2 13 26 PM" src="https://user-images.githubusercontent.com/24395931/166987027-e519fa80-4e64-4c17-a086-33d8328f2461.png">

Reply:
One option is that we just disable this warning repo-wide since we use it quite a bit

Reply:
Ahhh I think that'd be a good idea. We are always setting `v-html` from a trusted source, so globally disabling the rule seems justifiable.